### PR TITLE
Update UG with copy command

### LINKED ISSUE #121: Update UG for copy emails command

### PR DESCRIPTION
**Pull request by jeremiahlzz**
* Updated `README.md` site links with the actual site
* Removed redundant images
* Removed references to "addressbook"
* Rearranged command summary to follow the order which commands are explained
* Fixed minor grammatical errors
* Update add command summary to include weblink

Resolves #121, #119

### CODE REVIEW

**Review by marclamp — COMMENTED** [review]
LGTM :)

File: docs/UserGuide.md [hunk 1]
```diff
@@ -171,9 +172,28 @@ Examples:
...
+* Copied emails are in the form of `example1@email.com, example2@email.com, example3@email.com` and adheres to the format specified in [section 3.4 of RFC5322](https://tools.ietf.org/html/rfc5322#section-3.4).
+  * This makes it easy to paste into `Gmail`, `Outlook`, `Yahoo Mail`, etc.
+  <br>
+  <img src="images/copyToGmail.png" alt="result for 'copy t/tutorial-2'" style="border:4px solid #808080"/>
+
+Examples:
+* `copy` copies all emails
+* `copy t/tutorial-2` copies 3 emails (`johnsmith@example.com, irfan@example.com, royb@example.com` all with `tutorial-2` tag)
+* `copy t/tutorial-2 t/strong-student` copies 1 email (`royb@example.com` with both `tutorial-2` and `strong-student` tag) 
+
```
Comment (fy17ohhh) [inline docs/UserGuide.md:193]: Good explanation of the new command here:)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -171,9 +172,28 @@ Examples:
...
 ### Deleting a student : `delete`
 
-Deletes a specified student from the address book.
+Deletes a specified student.
```
Comment (fy17ohhh) [inline docs/UserGuide.md:196]: I also noticed that we have some phrases of'address book' in UG/DG, we should really remove them now as EduConnect is not an address book any more

**Review by fy17ohhh — APPROVED** [review]
LGTM:)

**Pull request merged**